File: src/main/java/net/greemdev/core/commands/ConsoleSayCommand.java
package net.greemdev.core.commands;

import github.scarsz.discordsrv.DiscordSRV;
import net.greemdev.core.util.CommandUtil;
import org.bukkit.ChatColor;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.jetbrains.annotations.NotNull;

public class ConsoleSayCommand implements CommandExecutor {

    @Override
    public boolean onCommand(@NotNull CommandSender sender, @NotNull Command cmd, @NotNull String commandLabel, @NotNull String[] args) {
        if (!CommandUtil.isAdmin(sender)) return true;
        if (CommandUtil.warnIfEmptyArgs(sender, args)) return true;

        DiscordSRV.getPlugin().getMainTextChannel().sendMessage("[Console] \u00BB " + String.join(" ", args)).queue();
        sender.getServer().broadcastMessage(ChatColor.GRAY + "[" + ChatColor.DARK_RED + ChatColor.BOLD + "Console" + ChatColor.GRAY + "]"
                + ChatColor.RESET + ": " + String.join(" ", args));

        return true;
    }
}

File: src/main/kotlin/net/greemdev/core/commands/ConsoleSayCommand.kt
package net.greemdev.core.commands

import github.scarsz.discordsrv.DiscordSRV
import net.greemdev.core.objects.CommandEvent
import net.greemdev.core.objects.CommandResult
import net.greemdev.core.util.*
import net.md_5.bungee.api.ChatColor
import org.bukkit.Bukkit
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender

public class ConsoleSayCommand: CommandExecutor {
    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<out String>): Boolean {
        return this.analyzeResult(this.executeCommand(CommandEvent.from(sender, command, label, args)))
    }

    private fun executeCommand(event: CommandEvent): CommandResult {
        if (!event.sender.isAdmin() || event.sender.warnIfEmptyArgs(event.args)) return CommandResult.unsuccessful()

        DiscordSRV.getPlugin().mainTextChannel.sendMessage("[Console] \u00BB " + event.args.join(" ")).queue()
        Bukkit.getServer().broadcastMessage(ChatColor.GRAY.toString() + "[" + ChatColor.DARK_RED + ChatColor.BOLD + "Console" + ChatColor.GRAY + "]"
                + ChatColor.RESET + ": " + event.args.joinToString(separator = " "))

        return CommandResult.successful()
    }

}